HCSocialSignIn/LandingViewController.swift

```diff
@@ -53,15 +55,13 @@ class LandingViewController: UIViewController {
         let redirectUrl = "http://hacknocraft.com/"
         let scope = ["r_basicprofile", "r_emailaddress"]
 
-        HCLinkedInManager.sharedInstance.login(viewController: self,
-                                               key: linkedInKey,
-                                               secret: linkedInSecret,
-                                               redirectUrl: redirectUrl,
-                                               scope: scope) { (success, _) in
-
-                                                if success {
-                                                    self.handleLinkedInLoginSuccess()
-                                                }
+        linkedInManager = HCLinkedInManager(key: linkedInKey, secret: linkedInSecret, redirectUrl: redirectUrl)
```

Comment:
I think we should define redirectURL as a constant. The user of the class should not worry about the redirect URL. 
Also, the linkedInManager should still be a singleton. There's no reason for the user to have to pass in the linkedin key and secret every time. The key and secret should be remembered by the shared instance.